svc-api/main.go

```diff
@@ -48,6 +48,21 @@ func main() {
 		}
 		basicAuth := r.Header.Get("Authorization")
 		var basicAuthToken string
+		// This if block added to fix issue odim-98
+		if basicAuth != "" {
+			var URLs_not_requiring_basic_auth = []string {"/redfish/v1","/redfish/v1/SessionService"}
```

Comment:
private variable names should start with a small letter. In Golang we use camel casing as a standard. Try to use meaningful short names.

Reply:
Done.